packages/app-project/components/ClassifyPage/components/ConnectWithProject/ConnectWithProjectContainer.js

```diff
@@ -18,7 +18,7 @@ function storeMapper (stores) {
 export default class ConnectWithProjectContainer extends Component {
   render () {
     const { projectName, urls } = this.props
-    const validUrls = urls.filter(urlObject => isValidUrl(urlObject.url))
+    const validUrls = urls.filter && urls.filter(urlObject => isValidUrl(urlObject.url))
```

Comment:
Should be:

```
const validUrls = urls && urls.filter(urlObject => isValidUrl(urlObject.url))
```

...otherwise the `should render null if not passed any urls` test throws `Cannot read property 'filter' of undefined`

Reply:
Cheers. I'm also seeing a props warning so I'll fix that too.
`Warning: Failed prop type: The prop `projectName` is marked as required in `ConnectWithProject`, but its value is `undefined`.
    in ConnectWithProject`

Should the code guard against the case where `urls` exists but isn't an array?
`const validUrls = urls && urls.filter && urls.filter(…)`.